ports/zephyr/main.c

```diff
@@ -81,6 +85,51 @@ void init_zephyr(void) {
     #endif
 }
 
+#if MICROPY_VFS
+STATIC int vfs_mount_and_chdir(mp_obj_t bdev, const char *mount_point_str) {
```

Comment:
looks like this comes from the stm32 port, so maybe it's a good idea to factor this out so it can be reused.  i'll look at doing that now.

Reply:
refactored in ed848553b408e14746f0ce96c8b20517bfdede35

Reply:
Thanks. I rebased and updated my patch to use the helper function.